src/shared/lib/handleTouchAndWheelEventsForChromeBug.ts

```diff
@@ -0,0 +1,43 @@
+// This is a temporary workaround until https://github.com/facebook/react/issues/14856 is fixed
+export function handleTouchAndWheelEventsForChromeBug(document: any) {
```

Comment:
Did you write the following code? Or is it coming from an existing solution? Just curious whether we need to review the following file.

Reply:
Yes, it's coming from an existing solution: https://github.com/facebook/react/issues/14856#issuecomment-478144231